docs/analog/commands/GetArchiveHours.md

```diff
@@ -0,0 +1,115 @@
+# GetArchiveHours
+
+Command to request/receive the archive data of hourly consumption.
+In the data field of the command, it is necessary to set the start date and hour for reading the archive.
+In case there is no data in the archive, a base value of `0xffffffff` will be provided.
+Since the length of the transmitted data from the sensor is limited, not all requested data will be transferred.
+
+
+## Request
+
+### Format
+
+| Size | Type                                   | Field                                 |
+| ---- | -------------------------------------- | ------------------------------------- |
+| `1`  | `byte`                                 | command id = `0x05`                   |
+| `1`  | `byte`                                 | command size = `4`                    |
+| `2`  | [packed date](../types.md#packed-date) | [start date](#start-date)             |
+| `1`  | `byte`                                 | [start hour](#start-hour)             |
+| `1`  | `byte`                                 | [number of values](#number-of-values) |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **start date**
+
+Start date for requested day pulse counter's values.
+<br>
+[See details](../types.md#packed-date).
+
+#### **start hour**
+
+Start hour for requested day pulse counter's values [`0..23`].
```

Comment:
would be nice to indicate that is a `../types.md#packed-hours` structure, but only as 1-5 first bits with start hour value, not full byte

Reply:
not sure about it
there are no more fields in this byte